File: administratorapp/lib/domain/entities/Usuario.dart
import 'package:flutter/material.dart';

class Usuario {
  final int usuarioid;
  final String email;
  final String nombre;
  final Text contrasena;

  Usuario({
    required this.usuarioid,
    required this.email,
    required this.nombre,
    required this.contrasena,
  });
}

File: api/Models/Cuenta/Usuario.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models.Cuenta;

public class Usuario
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Usuarioid { get; set; }

    public string Nombre { get; set; } = null!;

    public string Email { get; set; } = null!;

    public string Contrasena { get; set; } = null!;
}